workflow-fingerprint-collector/src/main/java/ca/on/oicr/pde/workflows/FingerprintCollectorWorkflow.java

```diff
@@ -34,11 +23,10 @@
     private String[] bamFiles;
     private String[] vcfFiles;
     private String[] gatkDirs;
-    private String studyName = "";
     private String dataDir;
-    private final String tempDir = "tempfiles/";
-    private final String gatkTmp = "temp";
-    private final String finDir  = "finfiles/";
+    private final static String TEMP_DIR = "tempfiles/";
+    private final static String GATK_TMP = "temp";
+    private final static String FINDIR  = "finfiles/";
     //private final int batchCount = 100; // Use for job batching, this many jobs
     private String gatkPrefix = "./";
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove the "gatkPrefix" field and declare it as a local variable in the relevant methods. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1450)